content/4. Deeltaken/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/1. Datatype-Declaratie.md

```diff
@@ -0,0 +1,82 @@
+---
+title:
```

Comment:
Titel toevoegen, niet als een list, maar gewoon als tekst.
Anders via een code editor aanpassen, gaat niet makkelijk in Obsidian